gpkit/substitution.py

```diff
@@ -179,8 +178,7 @@ def substitution(nomial, substitutions, val=None):
                 sub = VarKey(name=sub, **descr)
                 exps_[i] += HashVector({sub: x})
                 varlocs_[sub].append(i)
```

Comment:
@bqpd: when is the `elif isinstance(sub, Strings):` case invoked for substitution? Reason I ask is: should the Strings case do the same unit checking as the VarKey and Monomial cases?